request/graphql/schema/generate.go

```diff
@@ -1025,7 +1025,12 @@ func (g *Generator) genTypeFilterArgInput(obj *gql.Object) *gql.InputObject {
 						Type: operatorType,
 					}
 				} else { // objects (relations)
-					filterType, isFilterable := g.manager.schema.TypeMap()[genTypeName(field.Type, "FilterArg")]
+					fieldType := field.Type
+					if l, isList := field.Type.(*gql.List); isList {
+						// Filtering by inline array value is currently not supported
+						fieldType = l.OfType
+					}
+					filterType, isFilterable := g.manager.schema.TypeMap()[genTypeName(fieldType, "FilterArg")]
```

Comment:
todo: I think it might be worth chasing down why this failed in the way it did before merging this, it suggests a failing of the request validation that may hide additional (including future) bugs.